report/main.qmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "What is the best way to promote zero calories beverages?"
+format: 
+  pdf:
+    fontsize: 11pt
+editor: visual
+bibliography: ../reference/reference.bib
+author: Ivy Liu, Zefan Liu, Tom Tang
+---
+
+## Introduction
+
+Zero calorie beverages are healthier compared to sugared beverages because they contain less sugar and significantly fewer calories. Over the past decades, hospitals have been striving to promote zero-calorie beverages for a healthier lifestyle. This study employs five interventions to help increase such promotion. These interventions include a 10% price discount, a price discount combined with messaging that explains the reason for the discount, messaging displaying the caloric content in sugared beverages, messaging indicating the amount of exercise needed to burn off the calories in a sugared beverage, and messaging containing both caloric content and the amount of physical activity needed.
+
+The study aims to investigate whether these interventions help encourage people to choose zero calorie beverages over sugared beverages and if such effects are consistent across hospitals. The primary statistical question in this context is: "Is there any association between interventions and the increase in zero calorie beverage consumption or decrease in sugary beverage consumption?" The secondary statistical question is:"Does the effect of intervention differs by site?"
+
```

Comment:
I suggest adding one paragraph to talk about the organization for following sections.